src/lib/srocket.ts

```diff
@@ -62,11 +62,14 @@ export class Srocket<T, U> {
   /** Queue a message to send to the server, with "at-most-once" semantics. */
   send(message: U) {
     const data = encode(message);
+
+    // Convert buffer to Uint8Array
+    const uint8Data = new Uint8Array(data);
```

Comment:
(curious why this is needed now?)

Reply:
the conversion is actually needed because encode() returns a node.js Buffer, but our buffer array and WebSocket interface expect a Uint8Array. Without the conversion, typeScript gives an error since Buffer and Uint8Array are different types (even though they're similar under the hood). The error specifically mentions that Buffer isn't assignable to Uint8Array<ArrayBufferLike>

Reply:
That's strange, it shouldn't be returning a Node.js buffer. It's running in the browser. I think the type checking configuration might be a little borked

Reply:
Filed a ticket https://github.com/kriszyp/cbor-x/issues/120